### PR TITLE
Issue 121

### PR DESCRIPTION
**Pull request by diatomsRcool**

### CODE REVIEW

File: src/ontology/ecocore-edit.owl [hunk 1]
```diff
@@ -17,7 +17,7 @@ Import(<http://purl.obolibrary.org/obo/ecocore/imports/bfo_import.owl>)
 Import(<http://purl.obolibrary.org/obo/ecocore/imports/go_import.owl>)
 Import(<http://purl.obolibrary.org/obo/ecocore/imports/envo_import.owl>)
 Import(<http://purl.obolibrary.org/obo/ecocore/imports/uberon_import.owl>)
-Import(<http://purl.obolibrary.org/obo/ecocore/imports/ro_import.owl>)
+Import(<http://purl.obolibrary.org/obo/ro/releases/2020-07-21/ro.owl>)
```
Comment (matentzn) [inline src/ontology/ecocore-edit.owl:20]: This change seems wrong. The import should be as it was!

**Review by matentzn — CHANGES_REQUESTED** [review]

File: src/ontology/ecocore-edit.owl (hunk 1 above)
Comment (matentzn) [reply src/ontology/ecocore-edit.owl:20]: http://purl.obolibrary.org/obo/ecocore/imports/ro_import.owl

**Comment by matentzn** [comment]
Other than the import, the rest looks ok; I will just have to trust you :) 

Even if you don't use DOSDP tools, it is super useful to start building a few patterns for the most prevalent models in ecocore!

**Comment by matentzn** [comment]
Almost, but the import should be without a version: http://purl.obolibrary.org/obo/ecocore/imports/ro_import.owl

Its easiest to just change the text file, Rather than doing it in protege.

**Comment by diatomsRcool** [comment]
I must be crazy because it looks fixed to me

**Review by matentzn — APPROVED** [review]

**Pull request merged**